blocks/footer/footer.js

```diff
@@ -15,13 +29,68 @@ export default async function decorate(block) {
   if (resp.ok) {
     const html = await resp.text();
 
-    // decorate footer DOM
-    const footer = document.createElement('div');
-    footer.innerHTML = html;
+    // Create container for footer
+    const footerContainer = document.createElement('div');
+    footerContainer.classList.add('footer-container');
+    footerContainer.innerHTML = html;
+
+    const slogan = footerContainer.querySelector('div:nth-child(2) > p:nth-child(1)');
+    slogan.classList.add('footer-slogan');
+
+    footerContainer.querySelectorAll('ul')
+      .forEach((list, index) => {
+        list.classList.add(`footer-menu-list-${index}`);
+      });
+
+    // add horizontal line
+    const list = footerContainer.querySelector('ul');
+    const horizontalLine = document.createElement('hr');
+    list.after(horizontalLine);
+
+    const rightSide = footerContainer.querySelector('div:nth-child(3)');
```

Comment:
please make these links open in a new window as well.

Reply:
will do